azure-iot-device/azure/iot/device/iothub/pipeline/pipeline_stages_iothub_mqtt.py

```diff
@@ -80,6 +80,64 @@ def _execute_op(self, op):
                 ),
             )
 
+        elif (
+            isinstance(op, pipeline_ops_base.UpdateSasTokenOperation)
+            and self.pipeline_root.connected
```

Comment:
>self.pipeline_root.connected [](start = 16, length = 28)

My understanding is that if we are not connected, this op gets passed like any other unhandled op, right?

If so, a comment here to that effect could bring clarity, it took me a while to realize this.